src/templates/teamMemberTemplate.js

```diff
@@ -1,162 +1,107 @@
-        heroSubtitle={role}
-        heroClass="team-member__hero"
-        pageTitle={`CivicActions | ${name}`}
-        ogImage={image.childImageSharp.fluid.src}
-        teamImage={image}
-        location={memberLocation}
-        personal_pronouns={personalPronouns}
-        social={social}
-        path={frontmatter.path}
-        urlObject={this.props.location}
-        audioFile={nameAudioFile}
-        pronunciation={namePronunciation}
-      >
-        <div className="team_member__specs--wrapper">
-          <section className=" section usa-grid team-member__specs">
-            <div className="study__tech-specs ">
-              <span className="study__tech-specs__title"> Specialties </span>
-              <div> {specs} </div>
-            </div>
-          </section>
-        </div>
-        <section className="section text-container team-member__text">
-          <div dangerouslySetInnerHTML={{ __html: html }} />
-          {// Just show quote if quote exists
-          quote && existy(quoteName) ? (
-            <Blockquote quote={quote} quote_source={quoteName} />
-          ) : (
-            quote
-          )}
+  })
+  return (
+    <GeneralLayout
+      heroTitle={member.Name}
+      heroSubtitle={member.Role}
+      heroClass="team-member__hero"
+      pageTitle={`CivicActions | ${member.Name}`}
+      ogImage={member.Image.childImageSharp.fluid.src}
+      teamImage={member.Image}
+      location={member.Location}
+      personal_pronouns={member.Personal_Pronouns}
+      social={member.Social}
+      path={member.Path}
+      urlObject={location}
+      audioFile={member.Audio}
+      pronunciation={member.Pronunciation}
+    >
+      <div className="team_member__specs--wrapper">
+        <section className=" section usa-grid team-member__specs">
+          <div className="study__tech-specs ">
+            <span className="study__tech-specs__title"> Specialties </span>
+            <ul className="hero__specs--list">
+              {member.Specialty.map((spec, index) => (
+                <li className="hero__specialties" key={index}>
+                  {spec.Specialty}
+                </li>
+              ))}
+            </ul>
+          </div>
         </section>
-
-        {// Just show medium posts if medium posts exist
-        mediumPostsList ? (
-          <section className="section section__recent-posts team team-member__posts">
-            <div className="usa-grid">
-              <SectionTitle title="Authored articles" />
-              {mediumPostsList}
-            </div>
-          </section>
+      </div>
+      <section className="section text-container team-member__text">
+        <Markdown source={member.Body} escapeHtml={false} />
+        {member.Quote ? (
+          <Blockquote quote={member.Quote} quote_source={member.Name} />
         ) : (
-          mediumPostsList
+          <div></div>
         )}
-      </GeneralLayout>
-    )
-  }
+      </section>
+    </GeneralLayout>
+  )
```

Comment:
@Yodart are we no longer showing medium posts for team members? Fwiw I no longer see them in the current site so maybe the answer is yes?